helm-charts/common/speecht5/values.yaml

```diff
@@ -36,6 +36,9 @@ securityContext:
     - ALL
   seccompProfile:
     type: RuntimeDefault
+  # Init container sets the downloaded model dir to be group writable, so that container
+  # can keep its lock file there. This relies on both containers using the same group ID.
+  runAsGroup: 0
```

Comment:
These are now under (main pod) `securityContext`.

However, to be shared between pod containers, they would need to be under `podSecurityContext` (at line 26).

Reply:
done